src/overlays/actors/ovl_Obj_Lightswitch/z_obj_lightswitch.c

```diff
@@ -271,8 +235,8 @@ void ObjLightSwitch_Asleep(ObjLightswitch* this, GlobalContext* globalCtx) {
 
     Math_StepToS(&this->edgeRotSpeed, -0xAA, 0xA);
     this->edgeRot += this->edgeRotSpeed;
-    this->colorR = (this->colorShiftTimer * 0x140) + 0x26C0;
-    this->colorG = (this->colorShiftTimer * 0x1A0) + 0x1F40;
+    this->colorR = (this->colorShiftTimer * (5 << 6)) + (155 << 6);
+    this->colorG = (this->colorShiftTimer * 0x1A0) + (125 << 6); // 0x1A0 = 6 << 6
```

Comment:
I don't really understand why these colors are shifted like this, but I'm curious why this one here is left as a comment instead of being converted like the rest.

Reply:
I knew this question was coming and for some reason, IDO optimizes the code to non-matching when I use `(6 << 6)`, but I've left them in for other places as it's easier to see how the colour values are being set/used
![image](https://user-images.githubusercontent.com/47598039/125374238-f3042980-e3c9-11eb-9939-e2634988fff4.png)

Reply:
Actually (6 << 6) isn't `0x1A0`... my bad. I'll fix it

Reply:
Hex comments are fine. Leave bitshifts like this in decimal.